conda.recipe/build.sh

```diff
@@ -8,15 +8,12 @@ BLD_DIR=`pwd`
 SRC_DIR=$RECIPE_DIR/..
 pushd $SRC_DIR
 
-# X.X.X.dev.YYYYMMDD builds
-if [ -e using_tags.txt ]; then
-    version=`git describe --tags`
-else
-    version=`$PYTHON scripts/get_bump_version.py`
-fi
-
-date=`date "+%Y%m%d"`
+version=`$PYTHON scripts/get_bump_version.py`
```

Comment:
For some reason I thought we were doing away with "date based" dev build identifiers. If so, can't `get_bump_version.py` be done away with?